src/System.Private.CoreLib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -65,6 +65,27 @@
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Unix\System.Native\Interop.Write.cs"/>
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="$(MSBuildThisFileDirectory)System\Collections\DictionaryEntry.cs"/>
+    <Compile Include="$(MSBuildThisFileDirectory)System\Collections\Generic\ICollection`1.cs"/>
```

Comment:
Back tick in the filenames breaks source code indexing (see https://github.com/dotnet/corefx/issues/8608). Either use no arity suffix like majority of the filenames in CoreCLR and CoreFX, or
`*_1.cs` style like filenames in corefx immutable collections; or `*OfT.cs` style like a few filenames here and there.